webapp/components/admin_console/developer_settings.jsx

```diff
@@ -48,13 +48,13 @@ export default class DeveloperSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.service.testingTitle'
-                            defaultMessage='Enable Testing: '
+                            defaultMessage='Enable Load Testing: '
```

Comment:
Not sure what this setting actually does--is it enable just the /loadtest command? Does it enable other functions? The problem is /loadtest isn't for load testing only, it does functional testing as well.